rlxnix/dataset.py

```diff
@@ -125,10 +125,13 @@ def _scan_traces(self):
         continuous_type = type_map[self._relacs_nix_version][DataType.Continuous]
 
         for da in self._block.data_arrays:
-            if event_type in da.type:
+            check_event_type = any([True if t in da.type else False for t in event_type ])
```

Comment:
``check_event_type`` sounds active to me, as if it was a function but it is actually only a boolean --> rename to ``is_event_type`` ?